workflow/rules/trim_3prime.smk

```diff
@@ -0,0 +1,88 @@
+if is_3prime_experiment:
+
+    # Rule cutadapt1 checks and removes poly-A tails and sequence qualilty score <20.
+    # reasoning behind parameters:
+    #   * `-m 20`:
+    #   * Discards any read under 20bp by -m 20
+    #   * `-O 20 -a ""polyA=A{20}"" -a ""QUALITY=G{20}"" -n 2`:
+    #   * Removes reads having polyA (18bp) or polyG (18bp) and repeat the process twice by -n 2
+    # In short, it removes reads that are predominantly polyA or polyG.
+
+    # Rule cutadapt2 checks if reads are obtained from nextseq and removes adapters specific to nextseq
```

Comment:
This option applies a special trimming mode, but it always does this. It would also do it for non-nextseq reads. It is fine to keep the code like this (as we want the vendor key in the config which makes it clear), but the comment should not claim that we really check for nextseq reads here.

Reply:
Noted. Updated the comment section